cmd/configure/createprofile/createprofile.go

```diff
@@ -0,0 +1,64 @@
+package createprofile
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var CreateProfileCmd *cobra.Command
+
+func init() {
+	CreateProfileCmd = &cobra.Command{
+		Use:   "create-profile",
+		Short: "Create profile",
+		Long:  "Creates a new profile. Use this to create a profile with a name, host, and token.",
+		RunE:  createProfileHandler,
+	}
+
+	_ = CreateProfileCmd.Flags().String("name", "", "Name of profile.")
+	_ = CreateProfileCmd.Flags().String("host", "", "Host for profile.")
+	_ = CreateProfileCmd.Flags().String("token", "", "Token for profile.")
+
+	// Required flags
+	_ = CreateProfileCmd.MarkFlagRequired("name")
+	_ = CreateProfileCmd.MarkFlagRequired("host")
+	_ = CreateProfileCmd.MarkFlagRequired("token")
+}
+
+func createProfileHandler(cmd *cobra.Command, args []string) error {
+	name, err := cmd.Flags().GetString("name")
+	if err != nil {
+		return err
+	}
+
+	host, err := cmd.Flags().GetString("host")
+	if err != nil {
+		return err
+	}
+
+	token, err := cmd.Flags().GetString("token")
+	if err != nil {
+		return err
+	}
+
+	v := viper.GetViper()
+
+	if v == nil {
+		return fmt.Errorf("unable to access viper instance")
+	}
+
+	newProfile := map[string]interface{}{
+		"host":  host,
+		"token": token,
+	}
+
+	if v.Sub("profiles").InConfig(name) {
```

Comment:
To check if a profile already exists keep in mind that the config file structure is the following one:
```ini
[profile1]
host=foo
token=foo

[profiel2]
host=foo
token=foo
```

By writing it like this then the assumption on the configuration file is different, it should look like:
```ini
[profiles]
[profile1]
token=foo
host=foo

[profile2]
token=foo
host=foo
```

I would suggest to keep it as is and do not put profile under a "profiles" key